### ISSUE TITLE
finetune

### ISSUE THREAD

**Issue by alicera** [xingyizhou/CenterNet#875]
when finetune the model , how to design the learning rate?
